tracer/test/Datadog.Trace.DuckTyping.Tests/Fields/ReferenceType/ReferenceTypeFieldTests.cs

```diff
@@ -105,6 +105,7 @@ public void StaticFields(string obscureObjectName)
             Assert.Equal("60", duckAbstract.PublicStaticReferenceTypeField);
             Assert.Equal("60", duckVirtual.PublicStaticReferenceTypeField);
 
+            duckInterface.PublicStaticReferenceTypeField = "20";
```

Comment:
Would it be better to set these in the constructor instead, so they're always reset when the tests start? Net effect is the same 🤷‍♂️